lock-api/src/main/java/com/palantir/lock/LockRequest.java

```diff
@@ -48,8 +50,21 @@
 @Immutable public final class LockRequest implements Serializable {
     private static final long serialVersionUID = 0xf6c12b970b44af68L;
 
+    private static final AtomicReference<TimeDuration> DEFAULT_LOCK_TIMEOUT = new AtomicReference<>(
+            SimpleTimeDuration.of(120, TimeUnit.SECONDS));
```

Comment:
yeah that's be ideal - unfortunately this project does not depend on atlasdb-client